src/ctap/hid/receive.rs

```diff
@@ -577,13 +578,13 @@ mod test {
             Ok(None)
         );
         for seq in 0..0x7F {
-            timestamp += delay;
+            timestamp = timestamp + delay;
```

Comment:
Curious why this is necessary, would we need to send a PR to embedded_time to allow this?

Reply:
Compiler: cannot use `+=` on type `embedded_time::Instant<clock::TestClock>`